pyproject.toml

```diff
@@ -10,7 +10,7 @@ authors = [{ name = "Dan Foreman-Mackey", email = "foreman.mackey@gmail.com" }]
 requires-python = ">=3.8"
 license = { file = "LICENSE" }
 urls = { Homepage = "https://github.com/dfm/jax-finufft" }
-dependencies = ["jax", "jaxlib"]
+dependencies = ["jax", "numpy"]
```

Comment:
I don't think we should need this here! `numpy` is a dependency of `jax`. I think it's probably right to remove `jaxlib`, because that can cause problems, but we'll need to manually install `jaxlib` when running the tests.

Reply:
> numpy is a dependency of jax

I was thinking that it was good practice to specify direct dependencies even if they're also transitive, but I agree it's a no-op in this case! Really I just wanted the readme to be consistent with the `pyproject.toml` dependencies.

Makes sense about jaxlib, I'll update the CI!